Libplanet/Blockchain/BlockChain.cs

```diff
@@ -78,21 +78,24 @@ public class BlockChain<T>
         /// be used for that.</param>
         /// <param name="render">Defines whether to render actions on this
         /// <see cref="BlockChain{T}"/>.  Turned on by default.</param>
+        /// <param name="stateStore"><see cref="IStateStore"/> to store states.</param>
         /// <exception cref="InvalidGenesisBlockException">Thrown when the <paramref name="store"/>
         /// has a genesis block and it does not match to what the network expects
         /// (i.e., <paramref name="genesisBlock"/>).</exception>
         public BlockChain(
             IBlockPolicy<T> policy,
             IStore store,
             Block<T> genesisBlock,
-            bool render = true
+            bool render = true,
+            IStateStore stateStore = null
```

Comment:
How about move `stateStore` parameter next to `store` parameter?
```csharp
public BlockChain(
    IBlockPolicy<T> policy,
    IStore store,
    IStateStore stateStore = null,
    Block<T> genesisBlock,
    bool render = true
)
```

Reply:
And this change should be written in `CHANGES.md`.

Reply:
> How about move `stateStore` parameter next to `store` parameter?
> 
> ```cs
> public BlockChain(
>     IBlockPolicy<T> policy,
>     IStore store,
>     IStateStore stateStore = null,
>     Block<T> genesisBlock,
>     bool render = true
> )
> ```

The line, `IStateStore stateStore`, can not be there because it has a default value (i.e. `null`). But I believe it seems better to remove default value and to change parameter order change. I will try to change it and will leave a comment on this thread after it's done.



> And this change should be written in `CHANGES.md`.

You're right. I will add it.

Reply:
I moved the parameter next to `store` parameter 48928d2 . Also updated changelog dadc315 .